.eslintrc.js

```diff
@@ -24,7 +24,8 @@ module.exports = {
                 'plugin:@cspell/recommended',
                 'plugin:@angular-eslint/ng-cli-compat',
                 'plugin:@angular-eslint/ng-cli-compat--formatting-add-on',
-                'plugin:@angular-eslint/template/process-inline-templates'
+                'plugin:@angular-eslint/template/process-inline-templates',
```

Comment:
FYI we can easily prevent checking spec files (or other files which we don't want to include) if we do something like suggested here: https://github.com/Alfresco/alfresco-ng2-components/pull/8942#discussion_r1340049673

But we can leave it for now.

Reply:
those functions sometimes also good to document, to give other devs ideas what they are doing, anyway we can review later